__tests__/app.test.js

```diff
@@ -539,3 +539,13 @@ describe("9. DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+describe("10. GET /api", () => {
+  test("200: responds with an object containing all the endpoints available", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.endpoints).toBeInstanceOf(Object);
```

Comment:
Can test how many endpoints there are in the object. Also can test specific description.